highway_env/envs/common/observation.py

```diff
@@ -107,6 +107,7 @@ def __init__(self, env: 'AbstractEnv',
                  order: str = "sorted",
                  normalize: bool = True,
                  clip: bool = True,
+                 see_behind: bool = False,
```

Comment:
Last request: can you please also add the `see_behind` parameter to the docstring below?